homeassistant/components/lifx/light.py

```diff
@@ -455,20 +458,34 @@ def __init__(self, bulb, effects_conductor):
         self.postponed_update = None
         self.lock = asyncio.Lock()
 
+    def get_mac_addr(self):
+        """Increment the last byte of the mac address by one for FW>3.70."""
```

Comment:
This sounds like something that should be handled by the library.

Reply:
The library has no need to know this. It's a Home Assistant thing, because it's only needed to create connections to multiple integrations in the Device Registry.

Reply:
If the library defined bulb instance has an attribute called `mac_addr` and it no longer returns a mac address, that should be fixed in the library and not in Home Assistant.